src/main/java/core/basesyntax/MainApp.java

```diff
@@ -13,4 +16,16 @@
  */
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        List<Machine> machines = new ArrayList<>();
+        machines.add(new Track());
+        machines.add(new Bulldozer());
+        machines.add(new Excavator());
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
```

Comment:
first we should start all machines and then stop them. Read task closely)

Reply:
thank you, I separated them in two loops (as it was before, and I clarified that at Q&A session)